### PR TITLE
feat(TDI-46568): bump component-runtime to 1.36.0 in Studio 8

### PR DESCRIPTION
**Pull request by undx**
*Related PR*
- https://github.com/Talend/studio-se-master/pull/455
- https://github.com/Talend/tcommon-studio-se/pull/4518

### CODE REVIEW

File: main/plugins/org.talend.designer.maven.repo.tck/pom.xml [hunk 1]
```diff
@@ -11,6 +11,18 @@
 	<packaging>eclipse-plugin</packaging>
 	<properties>
 		<jackson.version>2.11.4</jackson.version>
+		<component-runtime.version>1.36.0-SNAPSHOT</component-runtime.version>
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.designer.maven.repo.tck/pom.xml:14]: component-runtime.version is already defined in https://github.com/Talend/studio-se-master/blob/master/talend.studio.parent.pom/pom.xml

Could you please help update https://github.com/Talend/studio-se-master/blob/master/talend.studio.parent.pom/pom.xml instead?

Comment (undx) [reply main/plugins/org.talend.designer.maven.repo.tck/pom.xml]: Done

File: main/plugins/org.talend.designer.maven.repo.tck/pom.xml [hunk 2]
```diff
@@ -11,6 +11,17 @@
 	<packaging>eclipse-plugin</packaging>
 	<properties>
 		<jackson.version>2.11.4</jackson.version>
```
Comment (ozhelezniak-talend) [inline main/plugins/org.talend.designer.maven.repo.tck/pom.xml:13]: We won't update jackson for the studio?
We updated it for the connectors...

**Review by ozhelezniak-talend — COMMENTED** [review]
Is it just a regular bump?
We added a lot of dependencies.

File: main/plugins/org.talend.designer.maven.repo.tck/pom.xml (hunk 2 above)
Comment (undx) [reply main/plugins/org.talend.designer.maven.repo.tck/pom.xml:13]: We match here the current connectors jackson's version but in fact I didn't saw that connector's were bumped! :-( 
Will bump it also...

Comment (undx) [reply main/plugins/org.talend.designer.maven.repo.tck/pom.xml:13]: @bhe-talendbj, will this bump (to `2.12.1`) impact studio features, it's it safe for you?

Comment (bhe-talendbj) [reply main/plugins/org.talend.designer.maven.repo.tck/pom.xml:13]: Should be safe, this plugin is only used by tck components.

File: main/plugins/org.talend.designer.maven.repo.tck/pom.xml [hunk 3]
```diff
@@ -109,6 +120,251 @@
...
+			<artifactId>component-api</artifactId>
+			<version>${component-runtime.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.talend.sdk.component</groupId>
+			<artifactId>component-runtime-design-extension</artifactId>
+			<version>${component-runtime.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.talend.sdk.component</groupId>
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.designer.maven.repo.tck/pom.xml:314]: It seems that we do not need to add these jars required by component servers. it should be updated in https://github.com/Talend/tcommon-studio-se/blob/master/main/plugins/org.talend.designer.maven.repo.tcksdk/pom.xml


For TCK, 

We splitted as follows

org.talend.designer.maven.repo.tcksdk -- jars required by tck component server
org.talend.designer.maven.repo.tck -- commons jars required by tck components
org.talend.studio.components.di.xxx , for example org.talend.studio.components.di.neo4j -- plugins for tck component - neo4j



org.talend.designer.maven.repo.tcksdk
https://github.com/Talend/tcommon-studio-se/blob/master/main/plugins/org.talend.designer.maven.repo.tcksdk/pom.xml

org.talend.designer.maven.repo.tck
https://github.com/Talend/tcommon-studio-se/blob/master/main/plugins/org.talend.designer.maven.repo.tck/pom.xml

org.talend.studio.components.di.neo4j
https://github.com/Talend/tdi-studio-ee/tree/master/main/plugins/org.talend.studio.components.di.neo4j

org.talend.studio.components.di.azuredlsgen2
https://github.com/Talend/tdi-studio-ee/blob/master/main/plugins/org.talend.studio.components.di.azuredlsgen2/pom.xml
....

File: main/plugins/org.talend.designer.maven.repo.tck/pom.xml (hunk 2 above)
Comment (undx) [reply main/plugins/org.talend.designer.maven.repo.tck/pom.xml:13]: same for `maintenance/7.3`?

File: main/plugins/org.talend.designer.maven.repo.tck/pom.xml (hunk 3 above)
Comment (undx) [reply main/plugins/org.talend.designer.maven.repo.tck/pom.xml:314]: Hi @bhe-talendbj, 
Ok, it's more clear for me. In fact, I think there was a misunderstanding last month when we discussed about this... 
Just a few questions about this new layout:

Are those poms still relevant?
- https://github.com/Talend/tcommon-studio-se/blob/master/main/plugins/org.talend.designer.maven.tos/resources/tcompv1/pom.xml
- https://github.com/Talend/tcommon-studio-se/blob/master/main/plugins/org.talend.designer.maven.tos/resources/tacokit/pom.xml

Seems that we have some duplicated dependencies...

Why some connectors-**se** are located in `tdi-studio-ee` and not `tdi-studio-se`? 

Thanks a lot!

Comment (bhe-talendbj) [reply main/plugins/org.talend.designer.maven.repo.tck/pom.xml]: yes, the refactor is for EE only.... so for tos still the same as before...

**Review by ozhelezniak-talend — APPROVED** [review]
LGTM

**Review by ozhelezniak-talend — APPROVED** [review]
LGTM

**Pull request merged**